sections/footer.liquid

```diff
@@ -73,14 +73,36 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign image_size_2x = settings.brand_image.width | times: 2 | at_most: 5760 -%}
+                      <img
+                        srcset= "{{ settings.brand_image | image_url: width: settings.brand_image.width }}, {{ settings.brand_image | image_url: width: image_size_2x }} 2x"
+                        src="{{ settings.brand_image | image_url: width: 760 }}"
+                        alt="{{ settings.brand_image.alt | escape }}"
```

Comment:
I've noticed in another PR that we tend to use the `| escape` filter often for alt text but I'm not sure why 🤔